core-services/release-controller/_releases/release-ocp-4.10-arm64.json

```diff
@@ -13,9 +13,29 @@
     "tag":{"tagRef":{"name":"4.10"}}
   },
   "verify":{
-    "arm64":{
+    "aws-arm64":{
```

Comment:
Do these names have to have arm64 in them?

Reply:
yes, according to Dennis we need to specify we are running arm64 on aws.